Sources/PostgreSQL/Connection/PostgreSQLConnection+Query.swift

```diff
@@ -39,7 +39,29 @@ extension PostgreSQLConnection {
             }
         }
     }
-    
+
+    /// Executes a raw query with the given parameters returning each row to the supplied handler.
+    ///
+    ///     try conn.query("SELECT id, firstName FROM users WHERE firstName = $1", ["Billy"]) { row in
+    ///         print(row)
+    ///     }
+    ///
+    /// - parameters:
+    ///   - query: The raw query string to execute.
+    ///   - resultFormat: Desired `PostgreSQLResultFormat` to request from PostgreSQL. Defaults to `.binary`.
+    ///   - onRow: PostgreSQL row accepting closure to handle results, if any.
+    public func query(_ query: String, _ parameters: [PostgreSQLDataConvertible] = [], resultFormat: PostgreSQLResultFormat = .binary,
```

Comment:
This should be possible to do with:

```swift
conn.raw("<sql here>").bind(42).bind("hello").run()
```

That's why this redundant `query` method was removed. 👍

Reply:
But, this isn't scalable - for example, when you have ~20-30 parameters, right? Is it okay to keep this in the lib so that folks can simply execute a query with some parameters like many other drivers?